pkg/store/helpers.go

```diff
@@ -75,6 +75,8 @@ func lt(mi, mj map[string]interface{}) bool {
 	case *big.Int:
 		mjs, _ := mj["pk0"].(*big.Int)
 		return mis.Cmp(mjs) < 0
+	case nil:
```

Comment:
it does not work, it is type casting, while `nil` is value

Reply:
nil - not value, nil- predictable identifier. This code work, try this:
```
func main() {
	fmt.Println(nilDetecting(nil))
}

func nilDetecting(in interface{}) bool {
	switch in.(type) {
	case nil:
		return true
	default:
		return false
	}
}
```
output: true

Reply:
nil - not value, nil- predictable identifier. This code work, try this:
```
func main() {
	fmt.Println(nilDetecting(nil))
}

func nilDetecting(in interface{}) bool {
	switch in.(type) {
	case nil:
		return true
	default:
		return false
	}
}
```
output: true